### PR TITLE
Now able to update PR comment/message

### PR DESCRIPTION
**Pull request by s1grun**

### CODE REVIEW

File: CompressImagesFunction/CompressImagesFunction.cs [hunk 1]
```diff
@@ -130,12 +130,14 @@ public static class CompressImagesFunction
...
             else if (didCompress)
             {
+                var update = compressImagesMessage.IsRebase;
                 logger.LogInformation("CompressImagesFunction: Successfully compressed images for {Owner}/{RepoName}", compressImagesMessage.Owner, compressImagesMessage.RepoName);
                 openPrMessages.Add(new OpenPrMessage
                 {
                     InstallationId = compressImagesMessage.InstallationId,
                     RepoName = compressImagesMessage.RepoName,
                     CloneUrl = compressImagesMessage.CloneUrl,
+                    Update = update,
```
Comment (javierron) [inline CompressImagesFunction/CompressImagesFunction.cs]: Here we can assign directly
```suggestion
                    Update = compressImagesMessage.IsRebase,
```

File: OpenPrFunction/PullRequest.cs [hunk 2]
```diff
@@ -29,12 +30,39 @@ public async Task<Pr> OpenAsync(GitHubClientParameters parameters, Settings sett
...
-            };
 
-            var result = await githubClient.PullRequest.Create(parameters.RepoOwner, parameters.RepoName, pr);
+            Octokit.PullRequest result;
+            if(update) {
+                //get PR number
+                var allPrs = await githubClient.PullRequest.GetAllForRepository(parameters.RepoOwner, parameters.RepoName);
+                
+                var prNumber = 0;
+                foreach(var pr in allPrs) {
```
Comment (javierron) [inline OpenPrFunction/PullRequest.cs:40]: Use Linq syntax to search in list
```
var pr = allPrs.First( pr => pr.State == ItemState.Open && pr.Head.Sha == branch.Commit.Sha);

if (pr == null)  { //throw}
```

**Review by javierron — COMMENTED** [review]
:partying_face:

File: OpenPrFunction/PullRequest.cs (hunk 2 above)
Comment (s1grun) [reply OpenPrFunction/PullRequest.cs:40]: I tried to implement it like this but wasn't able to make it work ....

**Pull request merged**